tests/test_string.py

```diff
@@ -38,7 +43,15 @@ def test_set(self):
         self.assertTrue(self.r.set(self.k1, self.v1))
         v1 = self.r.get(self.k1)
         self.assertEqual(self.v1, v1, '{} != {}'.format(v1, self.v1))
-
+    
+    def test_setbit(self):
+        ret = self.r.setbit(self.bitKey, self.bitPos1, self.bitVal1)
+        self.assertTrue(ret==0 or ret==1)
+
+    def test_getbit(self):
+        ret = self.r.getbit(self.k1, self.bitPos1)
```

Comment:
Before start a new unit test, the data will be cleared after the last test, so here you should `setbit` first, then do `getbit`.